.circleci/config.yml

```diff
@@ -57,6 +57,18 @@ jobs:
             - run: bundle exec rake test:performance
             - run: MOCHA_GENERATE_DOCS=1 bundle install --gemfile=<< parameters.gemfile >>
             - run: MOCHA_GENERATE_DOCS=1 GOOGLE_ANALYTICS_WEB_PROPERTY_ID=dummy rake yardoc
+  lint:
+    docker:
+      - image: ruby:2.2
```

Comment:
Set as 2.2 since that's the target Rubocop version, but I guess latest works as well?

Reply:
I'm struggling to get my head around whether it makes sense to run linting against the latest Ruby version or Rubocop's `TargetRubyVersion`, so for now I've stuck with the latter. However, I'm willing to be persuaded to change it to the latest Ruby version.